efel/pyfeatures/pyfeatures.py

```diff
@@ -140,6 +141,45 @@ def strict_burst_number() -> np.ndarray:
     return np.array([burst_mean_freq.size])
 
 
+def all_burst_number():
+    """The number of all the bursts, even if they have a single AP.
+
+    Instead of relying on burst_mean_freq, we split the ISIs into two groups,
+    and count the number of large ISIs. If there are no distinct two groups,
+    there are no bursts. The groups may not be even, which would correspond to
+    bursts with equal number of APs.
+    """
+    stim_start = _get_cpp_data("stim_start")
+    stim_end = _get_cpp_data("stim_end")
+    peak_times = get_cpp_feature("peak_time")
+    if peak_times is None:
+        return np.array([0])
+
+    peak_times = peak_times[(peak_times > stim_start) & (peak_times < stim_end)]
+    isis = np.diff(peak_times)
+
+    from sklearn.cluster import KMeans
```

Comment:
Could you put it at the top of the file and then add `scikit-learn>=X.Y.Z` to `setup.py`, where X.Y.Z is the version you tested?